src/components/pages/Car/CarsList.jsx

```diff
@@ -30,11 +30,12 @@ const CarsList = ({ cars }) => (
         customLeftArrow={<CustomArrow direction="left" />}
         infinite
       >
-        {cars.map((car) => (
+        {cars.map((car, index) => (
+
           <Link to={`/items/${car.id}`} key={car.id} className="link-to-detail">
             <div className="car-item">
               <div className="car-image">
-                <img src={car.photo} alt={car.name} />
+                <img src={images[index]} alt={car.name} />
```

Comment:
the images prop in the CarsList component is marked as required but its value is undefined.
<img width="1274" alt="1" src="https://github.com/ahmidou09/cruise-fleet-frontend/assets/122125958/fe20de03-82d5-400b-aa44-c7b4d790bc91">